apps/daffio/src/app/docs/guide-docs/components/guides-list/guides-list.component.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  Component,
+  Input,
+} from '@angular/core';
+
+import { DaffioGuideList } from '../../../shared/models/guide-list';
+
+@Component({
+  selector: 'daffio-guides-list',
```

Comment:
I think this component is more appropriately named `guide-nav`.

Reply:
changed